docs/EXTENDED-ARCHIVES.md

```diff
@@ -80,9 +75,9 @@ Archive layout
 The contents of an extended archive are the followings:
 
 - **REQUIRED** `.meta/manifest.json`
-- `.meta/publisher.svg`
-- `.meta/device.svg`
-- `.meta/image.svg`
+- `.meta/publisher(2x)?.[svg|png]`
```

Comment:
Probably better to split this out to separate files, as it currently makes it look like `.meta/publisher2x.svg` would be a valid option (and not everyone reading this will grok regexes).